fmriprep/run_workflow.py

```diff
@@ -121,8 +124,7 @@ def create_workflow(opts):
     # Set nipype config
     ncfg.update_config({
         'logging': {'log_directory': log_dir, 'log_to_file': True},
-        'execution': {'crashdump_dir': log_dir, 
-                      'remove_unnecessary_outputs': False}
+        'execution': {'remove_unnecessary_outputs': False}
```

Comment:
Sorry I misread the changelog. I thought you added the `'execution': {'remove_unnecessary_outputs': False}`. I'll open a new PR about this.